app/controllers/import_controller.rb

```diff
@@ -222,6 +221,22 @@ def validate_bool_field(field, value, row_ind)
     raise ValidationError.new(err_msg, row_ind)
   end
 
+  def validate_exclusive_race_field(field, row, col_num, row_ind)
+    value = validate_bool_field(field, row[col_num], row_ind)
+    return value if value.blank?
+
+    race_col_nums = [7, 8, 9, 10, 11, 101, 102, 103]
+    race_col_nums.each do |race_col|
+      next unless race_col != col_num
+
+      next unless value && row[race_col].to_s.downcase == 'true'
+
+      err_msg = "'#{VALIDATION[field][:label]}' cannot be true if any other race field is true"
```

Comment:
This doesn't appear to be working. I can import folks who have "Refused to Answer" as well as "Other" and other non-exclusive races set to true with no validation error.

Reply:
this code is no longer relevant as validation is now done in the `RaceValidator`